src/index.html

```diff
@@ -11,6 +11,21 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form data-qa="big" class="form-search form-search__big">
```

Comment:
please provide method attribute, accroding to readme

Reply:
added in solution 2